<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scheduler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonNew">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>New Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Schedule</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Schedule"/>
    <addaction name="actionOpen_Schedule"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Schedule"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport_xls"/>
    <addaction name="actionExport_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>Schedule</string>
    </property>
    <addaction name="actionAuto"/>
    <addaction name="actionAuto_Block"/>
    <addaction name="actionAuto_Assign"/>
    <addaction name="actionView_open_shifts"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSchedule"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Schedule">
   <property name="text">
    <string>New Schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Schedule">
   <property name="text">
    <string>Open Schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_xls">
   <property name="text">
    <string>Export..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSave_Schedule">
   <property name="text">
    <string>Save..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExport_as">
   <property name="text">
    <string>Export as..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionAuto_Assign">
   <property name="text">
    <string>Auto Assign</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAuto_Block">
   <property name="text">
    <string>Auto Block</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionView_open_shifts">
   <property name="text">
    <string>View open shifts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>menuBar</sender>
   <signal>hovered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>5</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateMenu()</slot>
 </slots>
</ui>
